### ISSUE TITLE
Loading autosaves edits them/changes the date & time on them

### ISSUE THREAD

**Issue by maarten99** [LoneGazebo/Community-Patch-DLL#8698]
_1. Mod version, and DirectX version if reporting a graphical issue (X.Y.Z, e.g. 1.4.6, DX 10 & 11):_
1.4.6, has been an issue since at least 1.2 I think

_3. Error description:_ 

When loading a regular save, they do not get edited, and the timestamp on the save stays the same.

When loading an autosave the timestamp changes, although I don't notice any actual ingame changes.

**Comment by ilteroi** [comment]
yeah for some reason the autosavee get overwritten after loading. seems to be benign
